mangaki/mangaki/utils/efa.py

```diff
@@ -0,0 +1,54 @@
+from mangaki.utils.common import RecommendationAlgorithm
+from django.conf import settings
+from mangaki.utils.chrono import Chrono
+from sklearn.decomposition import FactorAnalysis
+from scipy.sparse import csr_matrix
+import numpy as np
+
+
+class MangakiEFA(RecommendationAlgorithm):
```

Comment:
Please add a docstring with at least a link to relevant paper / explanation of the algorithm,